### PR TITLE
feat: add mongo

### PR DESCRIPTION
**Pull request by MalikEljaouadiWefox**

### CODE REVIEW

File: .rtx.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[tools]
```
Comment (IgorLaborieWefox) [inline .rtx.toml:1]: 🔨 **todo**: do not commit this file

File: rustainers/src/images/mongo.rs [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+/// # Example
+///
+/// ```rust, no_run
+/// # async fn run() -> anyhow::Result<()> {
+/// use rustainers::images::Mongo;
+///
+/// let default_image = Mongo::default();
+///
+/// let custom_image = Mongo::default()
+///        .with_tag("7.2");
```
Comment (IgorLaborieWefox) [inline rustainers/src/images/mongo.rs:23]: 🔨 **todo**: This version does not exist, use replace by `6`

**Review by IgorLaborieWefox — CHANGES_REQUESTED** [review]
Great stuff.

Still have few minor issues to fix.
Try to amend the 'feat: add mongo' commit, I want to rebase the PR instead of creating a merge commit.

File: .rtx.toml (hunk 1 above)
Comment (davidB) [reply .rtx.toml:1]: why?

**Review by IgorLaborieWefox — APPROVED** [review]

**Pull request merged**